apps/remix-ide-e2e/src/tests/terminal.test.ts

```diff
@@ -225,39 +225,70 @@ module.exports = {
-        .setValue('[data-id="modalDialogCustomPromp"]', 'https://rpc.archivenode.io/e50zmkroshle2e2e50zm0044i7ao04ym')
-        .modalFooterOKClick('basic-http-provider')
-        .click('[data-id="terminalClearConsole"]') // clear the console
-        .click('[data-id="listenNetworkCheckInput"]') // start to listen
-        .waitForElementContainsText('*[data-id="terminalJournal"]', 'from:', 200000)
-        .waitForElementContainsText('*[data-id="terminalJournal"]', 'to:', 200000)
-        .click('[data-id="terminalClearConsole"]') // clear the console
-        .waitForElementContainsText('*[data-id="terminalJournal"]', 'from:', 200000)
-        .waitForElementContainsText('*[data-id="terminalJournal"]', 'to:', 200000)
-        .click('[data-id="listenNetworkCheckInput"]') // stop to listen
-        .pause(30000)
-        .click('[data-id="terminalClearConsole"]') // clear the console
-        .pause(5000)
-        .click('[data-id="terminalClearConsole"]') // clear the console
-        .pause(20000)
-        .execute(function () {
-          return (document.querySelector('[data-id="terminalJournal"]') as any).innerText
-        }, [], function (result) {
-          browser.assert.equal(result.value, '', 'terminal log should be empty')
+  },
+
+  'Should listen on all transactions #group8': function (browser: NightwatchBrowser) {
+    const url = 'http://127.0.0.1:8545'
+    const identifier = 'Custom'
+    browser
+      .clickLaunchIcon('udapp') // connect to mainnet
+      .connectToExternalHttpProvider(url, identifier)
+      .openFile('contracts')
+      .openFile('contracts/1_Storage.sol')
+      .pause(4000)
+      .clickLaunchIcon('solidity')
+      .click({
+        selector: '*[data-id="compilerContainerCompileAndRunBtn"]',
+      })
+      .pause(10000)
+      .waitForElementNotPresent({
+        locateStrategy: 'xpath',
+        selector: "//*[@class='remix_ui_terminal_log' and contains(.,'to:') and contains(.,'from:')]",
+        timeout: 120000
+      })
+      .click({
+        selector: '[data-id="listenNetworkCheckInput"]',
+      }) // start to listen
+      .click({
+        selector: '*[data-id="compilerContainerCompileAndRunBtn"]',
+      })
+      .pause(10000)
+      .findElements(
+        {
+          locateStrategy: 'xpath',
+          selector: "//*[@class='remix_ui_terminal_log' and contains(.,'to:') and contains(.,'from:')]",
+          timeout: 120000,
+        }
+        , async (result) => {
+          if (Array.isArray(result.value) && result.value.length > 0) {
+            console.log('Found ' + result.value.length + ' transactions')
+            browser
+            .click({
+              selector: '[data-id="listenNetworkCheckInput"]',
+            })
+            .click({
+              selector: '*[data-id="terminalClearConsole"]',
+            })
+            .click({
+              selector: '*[data-id="compilerContainerCompileAndRunBtn"]',
+            })
+            .pause(10000)
```

Comment:
Same here

Reply:
yes it's because ganache-cli RPC is slow sending updates of transaction.